support/modules/pkg-install

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -e
+
+STATE="$1"
+FILES="$2"
+
+case "$STATE" in
+    ArtifactInstall)
+        RPMS=
+        DEBS=
+        for file in "$FILES"/files/*; do
+            case "$file" in
+                *.rpm)
+                    RPMS="$RPMS $file"
+                    ;;
+                *.deb)
+                    DEBS="$DEBS $file"
+                    ;;
```

Comment:
Nice and short - and could be shorter! :)
From my knowledge I think it is pretty rare to want to deploy both .deb and .rpm. You typically pick one format and stick with that, and I think deb is more common.

So I would simply rename to deb-install and remove the rpm parts (we can create another for that later).

Reply:
Just a FYI. 

From a Yocto perspective "rpm" and "ipk" are the supported package types that are part of the Yocto test suite. So in this context I would pick one of these over "deb".

Reply:
Since both appear to be important candidates, I split the module into one for each type, `deb-install` and `rpm-install`.